app/controllers/user_controller.py

```diff
@@ -7,8 +7,19 @@
 
 from app.models.User import User 
```

Comment:
Take a look on this [PR](https://github.com/Social-projects-Rivne/Cycling_admin/pull/3)
After it accepted you had to use  next construction to access to model
`from app.models.user import User`